api/envoy/service/status/v3/csds.proto

```diff
@@ -86,6 +86,11 @@ message ClientStatusRequest {
 
   // The node making the csds request.
   config.core.v3.Node node = 2;
+
+  // The flag to indicate whether xDS server needs to propagate the detailed
```

Comment:
Please change this comment to say something like this:

"""
If true, the server will not include the resource contents in the response (i.e., the generic_xds_configs.xds_config field will not be populated).
"""

The description may need to get more complicated if we also need to support the deprecated `PerXdsConfig` message, because it affects a number of different fields there.